specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/tspconfig.yaml

```diff
@@ -32,13 +32,15 @@ options:
     flavor: azure
     api-version: "2018-01-01"
   "@azure-tools/typespec-ts":
-    generateMetadata: true
-    generateTest: true
+    generateMetadata: false
+    generateTest: false
     azureSdkForJs: true
-    package-dir: "eventgrid-system-events"
+    experimentalExtensibleEnums: true
+    package-dir: "eventgrid-systemevents"
     packageDetails:
-      name: "@azure/eventgrid-system-events"
+      name: "@azure/eventgrid-systemevents"
```

Comment:
The package has been renamed in the azure-sdk-for-js repo https://github.com/Azure/azure-sdk-for-js/pull/32889